src/ripple/app/tx/impl/DepositPreauth.h

```diff
@@ -42,6 +42,12 @@ class DepositPreauth
     preclaim(PreclaimContext const& ctx);
 
     TER doApply () override;
+
+    // Interface used by DeleteAccount
+    static
+    TER
+    removeDepositPreauthFromLedger (Application& app, ApplyView& view,
```

Comment:
I would rename this `removeFromLedger` Rational: it's a class static function, it's should be called as `DepositPreauth::removeDepositPreauthFromLedger`. `DepositPreauth` is written twice, and there's no need for that.

Also, it's easier to use such functions in generic code, i.e. we can say `T::removeFromLedger` when `T` is a template arg, but there is no way to say `T::removeTFromLedger` without using macros. I'll provide a patch for this in the summary comment